### ISSUE TITLE
Out-of-date reference for URIs (Trac #34)

### ISSUE THREAD

**Issue by mnot** [httpwg/httpbis-issues#34]
```text
#!html
        <p>RFC2616 refers to RFC2396, RFC1630, RFC1738, and RFC1808, all of which have been obsoleted by RFC3986. </p>
        <p>If the reference is changed, it will important to remember to make a 
        decision about the "mark" production in the former (which are unreserved 
        characters) which have been moved to have reserved status in the latter.</p>
        <p>By referencing 2396, HTTP allows these characters unescaped in path 
        segments, etc. If 3986 is referenced as-is, they will be effectively 
        disallowed, thereby effectively making a number of existing HTTP URIs invalid.</p>
        <p>This includes the exclamation mark ("!"), asterisk ("*"), single-quote ("'"), 
        and open and close parentheses ("(" and ")").</p>
```

Migrated from https://trac.ietf.org/ticket/34
```json
{
    "status": "closed",
    "changetime": "2008-11-21T15:57:13",
    "_ts": "1227283033000000",
    "description": "{{{\n#!html\n        <p>RFC2616 refers to RFC2396, RFC1630, RFC1738, and RFC1808, all of which have been obsoleted by RFC3986. </p>\n        <p>If the reference is changed, it will important to remember to make a \n        decision about the \"mark\" production in the former (which are unreserved \n        characters) which have been moved to have reserved status in the latter.</p>\n        <p>By referencing 2396, HTTP allows these characters unescaped in path \n        segments, etc. If 3986 is referenced as-is, they will be effectively \n        disallowed, thereby effectively making a number of existing HTTP URIs invalid.</p>\n        <p>This includes the exclamation mark (\"!\"), asterisk (\"*\"), single-quote (\"'\"), \n        and open and close parentheses (\"(\" and \")\").</p>\n}}}",
    "reporter": "mnot@pobox.com",
    "cc": "",
    "resolution": "fixed",
    "time": "2007-12-20T05:07:02",
    "component": "p1-messaging",
    "summary": "Out-of-date reference for URIs",
    "priority": "",
    "keywords": "",
    "milestone": "05",
    "owner": "",
    "type": "design",
    "severity": ""
}
```

**Comment by mnot** [comment]
Trac update at 20080104T06:16:14: 
* mnot@pobox.com changed component from "" to "messaging"
* mnot@pobox.com changed milestone from "" to "unassigned"
* mnot@pobox.com changed owner from "" to ""

**Comment by mnot** [comment]
Trac update at 20080104T06:31:01: mnot@pobox.com changed version from "" to "00-draft"

**Comment by mnot** [comment]
Trac update at 20080205T03:23:09: fielding@gbiv.com commented:

> RFC3986 is the standard based on full knowledge of its effect on HTTP.
> IOW, the decision has already been made.
>

**Comment by mnot** [comment]
Trac update at 20081113T21:57:37: julian.reschke@gmx.de changed milestone from "unassigned" to "05"

**Comment by mnot** [comment]
Trac update at 20081113T22:27:56: 
* julian.reschke@gmx.de commented:

> Fixed in [374]:
> 
> Resolve #34: update to current URI spec, get rid of discussion of URI history (closes #34)


* julian.reschke@gmx.de changed resolution from "" to "fixed"
* julian.reschke@gmx.de changed status from "new" to "closed"

**Comment by mnot** [comment]
Trac update at 20081121T15:57:13: julian.reschke@gmx.de commented:

> From [423]:
> 
> make reason phrase for 414 consistent again (related to #34)
